test/helpers.test.js

```diff
@@ -0,0 +1,4 @@
+var convertToURI = require('../src/helpers.js')
```

Comment:
let's stick to es6 import syntax
`import { converToUri } from '../src/helpers'`